.github/workflows/cmake-ctest.yml

```diff
@@ -129,7 +129,7 @@ jobs:
       - name: Install Dependencies
         uses: ssciwr/doxygen-install@v1
         with:
-          version: "1.10.0"
```

Comment:
Are there significant difference between the "build_and_test" steps for the different platforms? Otherwise it'd be easy to have this be a single step instead of duplicated.